npm/npm_import.bzl

```diff
@@ -353,59 +215,35 @@ def npm_translate_lock(
         lifecycle_hooks: A dict of package names to list of lifecycle hooks to run for that package.
 
             By default the `preinstall`, `install` and `postinstall` hooks are run if they exist. This attribute allows
-            the default to be overridden for packages to run `prepare` for example,
+            the default to be overridden for packages to run `prepare`.
 
-            ```
-            lifecycle_hooks = {
-                "*": ["preinstall", "install", "postinstall"],
-                "@kubernetes/client-node": ["prepare"],
-                "fum@0.0.1": ["postinstall"],
-            }
-            ```
+            List of hooks are not additive. More specific name matches take precedence.
```

Comment:
Just "The most specific match wins." here same as `patch_args`?

Reply:
fixed throughout